component/core/src/test/java/org/exoplatform/social/core/jpa/test/InitContainerTestSuite.java

```diff
@@ -98,6 +95,7 @@
   MetadataActivityProcessorTest.class,
   ImageThumbnailServiceImplTest.class,
   RichEditorConfigurationServiceTest.class,
+  AttachmentActivityProcessorTest.class,
```

Comment:
The same class is added twice in two Test suites `InitContainerTestSuite` and `NoContainerTestSuite`

Please add it in a single place to not execute it twice in build time.

Reply:
done